examples/tensorflow/deepfm_tf/run.sh

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 rm -rf log* eval/ export/  checkpoint  model.ckpt* events.out* -rf graph.pbtxt
+pip install packaging
```

Comment:
Move the deps installing into the base image dockerfile.

Reply:
doesn't work, looks like we need to modify something else